src/test/java/step3/domain/car/CarsTest.java

```diff
@@ -0,0 +1,54 @@
+package step3.domain.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarsTest {
+
+    private static final int LOCATION = 0;
+    private static final int INTERVAL = 1;
+    private static final int CAR_COUNT = 3;
+    private static final int ROUND_COUNT = 3;
+
+    @DisplayName("전진, 정지 잘하는지 테스트")
+    @ParameterizedTest
+    @MethodSource(value = "generateGoInputs")
+    void goTest(int inputPower, int moveDistance) {
+        //given
+        List<Car> inputCars = createCars(LOCATION);
+        Cars cars = new Cars(inputCars, new StaticPower(inputPower));
```

Comment:
```suggestion
        Cars cars = new Cars(inputCars, () -> inputPower);
```
람다를 이용하면 StaticPower라는 테스트용 클래스를 만들지 않고 이 테스트를 진행 할 수 있어요! :)